pkg/proxy/winkernel/proxier.go

```diff
@@ -610,6 +692,15 @@ func NewProxier(
 			return nil, fmt.Errorf("source-vip flag not set")
 		}
 
+		if nodeIP.IsUnspecified() {
+			// attempt to get the correct ip address
+			klog.V(2).InfoS("node ip was unspecified.  Attempting to find node ip")
+			nodeIP, err = apiutil.ResolveBindAddress(nodeIP)
+			if err != nil {
+				klog.InfoS("failed to find an ip. You may need set the --bind-address flag", "err", err)
+			}
+		}
+
 		interfaces, _ := net.Interfaces() //TODO create interfaces
```

Comment:
This is a todo, it has a performance problem to iterate over the interfaces first and later over the addreses 
https://github.com/kubernetes/kubernetes/pull/103116

Reply:
I created https://github.com/kubernetes/kubernetes/issues/103588 to track.  fyi @sbangari